### PR TITLE
Use datNET namespace

### PR DESCRIPTION
**Pull request by awseward**

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -6,9 +6,9 @@
 open Fake
 open NuGetHelper
 open RestorePackageHelper
-open FSharpUtils.Fake.Config
+open datNET.Fake.Config
 
-DatNET.Targets.Initialize id
+datNET.Targets.Initialize id
```
Comment (mglodack) [inline build.fsx:11]: Are we going with `datNET` or `DatNET`?

Comment (awseward) [inline build.fsx:11]: That is completely open to discussion.

Comment (mglodack) [inline build.fsx:11]: I like `datNET`

**Comment by awseward** [comment]
@mglodack: This is the cause of the failing build

> error FS0039: The namespace or module 'FSharpUtil' is not defined

Do we wanna update the namespace piece by piece and keep things working, or just do this all at once and assume we'll come out fine on the other side?

**Comment by mglodack** [comment]
@awseward We should do them all at once. I think we have to change the config.fsx file as welll

**Comment by awseward** [comment]
Good catch, updated in https://github.com/datNET/fs-version-utils/pull/17/commits/20f2639af207250e614f70854bf52d32bfad147f

**Comment by mglodack** [comment]
+1

**Pull request merged**